yang/vyatta-protocols-frr-bgp-v1.yang

```diff
@@ -216,6 +216,25 @@ module vyatta-protocols-frr-bgp-v1 {
 		}
 	}
 
+	grouping parameters-multiple-paths {
+		container multiple-paths {
+			description "Forward packets over multiple paths";
+			configd:help "Forward packets over multiple paths";
+			leaf ebgp {
```

Comment:
Where possible we try to expose default values in the data model. In this case you could expand the range and add a default statement on both of these leaves, for example:
```
leaf ibp {
    type uint32 {
        range 1..256;
    }
    default 1;
    ...
}

Reply:
You certainly can't have a default outside the range.  So, even if you don't specify the default here, you need to expand the range to include the default.